### PR TITLE
fix: add ChallengeGame

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: src/contract/interfaces/IAdjudicationContract.ts [hunk 1]
```diff
@@ -1,7 +1,19 @@
 import { Bytes, BigNumber, List } from '../../types/Codables'
-import { Property } from '../../ovm'
+import { ChallengeGame, Property } from '../../ovm'
 
 export interface IAdjudicationContract {
+  /**
+   * Gets instantiated challenge game by gameId
+   * @param gameId
+   */
+  getGame(gameId: Bytes): Promise<ChallengeGame>
```
Comment (tkmct) [inline src/contract/interfaces/IAdjudicationContract.ts]: what if a game doesn't exist for the given id? returns null?

File: src/ovm/types.ts [hunk 2]
```diff
@@ -46,6 +46,20 @@ export class Property {
...
+ * ChallengeGame is a part of L2 dispute. It's instantiated by claiming property.
+ * Client can get game instance from Adjudicator Contract.
+ * https://github.com/cryptoeconomicslab/ovm-contracts/blob/53aeb7e121473a9d47dbc75d1f2ce4801a29b67e/contracts/DataTypes.sol#L13
+ */
+export class ChallengeGame {
+  constructor(
+    readonly property: Property,
+    readonly challenges: Bytes[],
+    readonly decision: boolean,
+    readonly createdBlock: Integer
```
Comment (tkmct) [inline src/ovm/types.ts]: I think blockNumber is `uint256` in CommitmentContract. isn't `createdBlock` inconsistent with those types?

Comment (syuhei176) [reply src/ovm/types.ts]: fixed

File: src/contract/interfaces/IAdjudicationContract.ts (hunk 1 above)
Comment (syuhei176) [reply src/contract/interfaces/IAdjudicationContract.ts]: can we throw exception if game is not found?

Comment (tkmct) [reply src/contract/interfaces/IAdjudicationContract.ts]: sure can.
I think it's good time to create custom error classes.

Comment (syuhei176) [reply src/contract/interfaces/IAdjudicationContract.ts]: ok! I added about exception to comment.

**Pull request merged**